src/client/containers/PullRequestDiff/__tests__/index.js

```diff
@@ -0,0 +1,69 @@
+import chai from 'chai'
+import {
+  getData,
+} from '../index'
+import { types } from 'ducks/pullrequests/actions'
+
+const expect = chai.expect
+
+describe('getData', () => {
+  it('full data', () => {
```

Comment:
Why is only successful case covered? Let cover all of them, no status, no users, no comments, no entities at all.